examples/lis3dsh_reader/lis3dsh_reader_main.c

```diff
@@ -49,7 +53,7 @@
 #include <nuttx/sensors/lis3dsh.h>
 
 /****************************************************************************
- * lis3dsh_reader_main
```

Comment:
for cleanup, I replaced "lis3dsh_reader_main"  with "Private Functions" which is a required string instead of adding a new "Private Functions" block.

Reply:
for cleanup, I replaced "lis3dsh_reader_main" with "Private Functions" which is a required string instead of adding a new "Private Functions" block.

Reply:
But it should be "Public Functions" here.

Reply:
I see that many source files keep both. Example:
```
/****************************************************************************
 * Public Functions
 ****************************************************************************/

/****************************************************************************
 * renew_main
 ****************************************************************************/

int main(int argc, FAR char *argv[])
```
So can we add `Public Functions` instead of replacing `lis3dsh_reader_main`?

Reply:
Yes.